.github/workflows/deploy.yml

```
@@ -50,43 +51,70 @@
-        run: |
-          docker push $ECR_REPOSITORY_URI:latest
-          docker push $ECR_REPOSITORY_URI:$IMAGE_TAG
-          echo "image=$ECR_REPOSITORY_URI:$IMAGE_TAG" >> $GITHUB_OUTPUT
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Amazon ECR Public
+        uses: docker/login-action@v3
+        with:
+          registry: public.ecr.aws
+          region: us-east-1
```

actionlint (1.7.7)

input "region" is not defined in action "docker/login-action@v3". available inputs are "ecr", "logout", "password", "registry", "username"

(action)